packages/v4/src/content/developer-resources/accessibility-development.md

```diff
@@ -98,3 +98,24 @@ The recommended interaction pattern for the modal components like the modal or p
 - For screen reader users, the other contents on the page should be hidden from the screen reader.
 
 Based on the [screen reader / browser combinations](https://www.patternfly.org/v4/get-started/accessibility-guide#testing) we use for testing, we recommend applying `aria-hidden="true"` to the parent wrapping element of the page contents. Note that the modal element of the component must not be a descendent of this element with `aria-hidden="true"` and should be included as a sibling to this element.
+
+### Scrollable elements
+Any elements with horizontal or vertical scroll need to be accessible via keyboard. It may be necessary to ensure every 
+container with an overflow rule and content which could possibly grow larger than its parent's height to have 
+`tabindex=0`. This will ensure that parent can capture browser focus and be scrolled using the arrow keys. Some browsers
+will automatically make any scrollable element focusable to ensure keyboard accessibility as long as they have a 
+non-negative tabindex.
+
+Additionally, when adding tabIndex in this scenario, it would also be beneficial to add an aria-label that describes the scrollable element.
```

Comment:
Slight tweak suggestion:

```suggestion
Additionally, when adding tabIndex in this scenario, it can also be beneficial to add an aria-label that describes the scrollable element, similar to how a heading describes a section of content.
```

Thoughts being that maybe it will paint a clearer picture for how to use `aria-label` for this use-case.